pilot/pkg/proxy/envoy/discovery.go

```diff
@@ -134,6 +134,16 @@ var (
 	// V2ClearCache is a function to be called when the v1 cache is cleared. This is used to
 	// avoid adding a circular dependency from v1 to v2.
 	V2ClearCache func()
+
+	// DebounceAfter is the delay added to events to wait
+	// after a registry/config event for debouncing.
+	// This will delay the push by at least this interval, plus
+	// the time getting subsequent events.
+	DebounceAfter time.Duration
```

Comment:
Will the events get squashed between two debounce events? (will comeup with a better name for debounce)

Reply:
'debounce' is the common name for this. 

Events should be squashed between debounce - but I'll keep testing. Pretty sure we'll not ignore 
a change, i.e. a push will happen and with a bound time.  What I'm not 100% sure yet is if there is a sequence that still results in 2 pushes for a sequence of changes - but that's not a regression, we already have that case.